filecoin-proofs/benches/aggregation.rs

```diff
@@ -59,6 +59,42 @@ fn bench_seal_inputs(c: &mut Criterion) {
     group.finish();
 }
 
+fn bench_sector_update_inputs(c: &mut Criterion) {
+    let params = vec![1, 16, 64, 256, 512, 1024];
+
+    let porep_id_v1_1: u64 = 8; // This is a RegisteredSealProof value
+
+    let mut porep_id = [0u8; 32];
+    porep_id[..8].copy_from_slice(&porep_id_v1_1.to_le_bytes());
+    assert!(!is_legacy_porep_id(porep_id));
+
+    let config = PoRepConfig::new_groth16(SECTOR_SIZE_32_GIB, porep_id, ApiVersion::V1_2_0);
+
+    let comm_r_old = [5u8; 32];
+    let comm_r_new = [6u8; 32];
+    let comm_d_new = [7u8; 32];
+
+    let mut group = c.benchmark_group("bench_sector_update_inputs");
+    for iterations in params {
+        group
+            .bench_function(format!("get-sector-update-inputs-{}", iterations), |b| {
+                b.iter(|| {
+                    for _ in 0..iterations {
+                        get_sector_update_inputs::<SectorShape32GiB>(
```

Comment:
Just noting that generating the inputs for a SnapDeal's 16x Groth16 proofs essentially just copies the three commitments `CommDNew, CommROld, CommRNew` 16 times (and should be constant-time), whereas SDR PoRep's input generation is more computationally expensive/variable.